src/main/resources/com/sumologic/sumobot/http_frontend/index.html

```diff
@@ -24,12 +24,16 @@
     <meta name="description" content="Sumobot-over-HTTP">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <script type="text/javascript" src="script.js"></script>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
 </head>
 <body>
-    <input title="message" id="message" name="message" type="text" />
-    <input type="submit" id="submit" value="Send" />
-    <div id="messages">
-        Messages:
+    <div class="page-container">
+        <div id="messages"></div>
+        <div class="inputs">
+            <input title="message" id="message-input" type="text" />
```

Comment:
A general remark - this is HTML, not XML. `<something />` is interpreted just as `<something>`. In case of an `input` it works as input doesn't have contents but it won't work in general. I'd avoid this tag style, just write `<input>`.

This applies to many other places as well.